surprise/__init__.py

```diff
@@ -47,4 +45,4 @@
     "model_selection",
 ]
 
-__version__ = get_distribution("scikit-surprise").version
```

Comment:
I don't think you want to hardcode 1.1.4 here, probably should go with the existing change